.lando.yml

```diff
@@ -5,7 +5,7 @@ config:
   database: mariadb
   composer_version: 2-latest
   via: nginx
-  xdebug: true
+  xdebug: false
```

Comment:
Curious: do you prefer to start projects w/o xdebug enabled and only enable it when needed?

Reply:
Yeah thinking more about FE devs here who typically don't use it. Depending on your local config, this can print out a lot of errors if enabled and not configured properly in your IDE. So this kind of avoids that by default.